linode/manager
# Change: Manifest Json

## Description

🚨 disclaimer: I have no idea if this works on Android

Updates `manifest.json` to get Cloud closer to a PWA. We're never really going to get there until we start looking at service workers, but this is a good start in terms of pinning the app to your phone's dashboard.

## Recommended Reading

[MDNs guide to manifest.jsons](https://developer.mozilla.org/en-US/docs/Web/Manifest)

[Why is the apple PNG icon not transparent?](https://makandracards.com/makandra/26757-do-not-use-transparent-pngs-for-ios-favicons)

[Why do we have both 512px and 192px PNGs?](https://developers.google.com/web/fundamentals/web-app-manifest)

## To test

1. Run Chrome Dev Tools lighthouse for PWAs. Compare this branch vs. develop

2. On either iOS or Android, [pin the webpage to your home screen](https://www.howtogeek.com/196087/how-to-add-websites-to-the-home-screen-on-any-smartphone-or-tablet/) and see how the icons look

## Preview

![unnamed](https://user-images.githubusercontent.com/7387001/65789529-8e5fd800-e12b-11e9-9cab-d3af7fe71a5e.jpg)
